README.md

```diff
@@ -13,34 +13,48 @@ This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.co
 ![](docs/demo-workflow.gif)
 
 
-## Logging
+## How does it work
```

Comment:
Generally there's inversion in questions (_How does it work_?) and no inversion in relative clauses (_I just asked how it works._). This caption is neither... but my general feeling would be that no inversion should be applied (_How it works_); see e.g. https://www.youtube.com/channel/UCELt4nocnWDEnYJmov4zqyA